cfgov/v1/atomic_elements/atoms.py

```diff
@@ -154,3 +155,33 @@ def clean(self, data):
             raise ValidationError("ImageBasicUrlAlt validation errors", params=error_dict)
         else:
             return data
+
+class AnchorLink(blocks.StructBlock):
+    link_id = blocks.CharBlock(required=False)
```

Comment:
Instead of making this a StructBlock, it should just subclass the CharBlock since this is the only block in it.